### PR TITLE
Schedule button to schedule

### LINKED ISSUE #134: Usability: Does the edit schedule button in the nav make sense?
We should probably ask some students, but I don't know if putting the edit button in the nav makes sense if the rest of the buttons pertaining to the schedule are actually near the schedule.

### PR DESCRIPTION
**Pull request by nchurchill**
I have made some changes to the way classes are added into the schedule. The schedule page has gone kaput  as noticed that there was no real difference between it and the home page other than that one could add classes and the other couldn't. We now show the x's on the home pages events so that they can be removed. these x's now open a modal that allows the user to ensure they want to cancel class.
fixes #134

### CODE REVIEW

File: src/Athena/Views/Home/Home.cshtml [hunk 1]
```diff
@@ -47,14 +47,50 @@ else
 
 }
 
+@Html.Partial("_CurrentScheduleModal", false)
+@Html.Partial("_UnmetDependencyError")
+
+<div id="modal2" class="modal">
```
Comment (nlowe) [inline src/Athena/Views/Home/Home.cshtml]: Let's make this ID a little more descriptive...

File: src/Athena/Views/Home/Home.cshtml [hunk 2]
```diff
@@ -47,14 +47,50 @@ else
...
+    <div class="modal-content">
+        <h4>Confirm Removal</h4>
+        <p>Please confirm that you would like to remove this class from your schedule.</p>
```
Comment (nlowe) [inline src/Athena/Views/Home/Home.cshtml]: Let's rephrase this:

> Are you sure you want to remove this course?

File: src/Athena/Views/Home/Home.cshtml [hunk 3]
```diff
@@ -47,14 +47,50 @@ else
...
+        <a href="#!" class="modal-action modal-close waves-effect btn">Cancel</a>
+    </div>
+</div>
+
 @section scripts {
     @if (User.Identity.IsAuthenticated)
     {
         <script type="text/javascript" src="@Url.Content("~/schedule.js")"></script>
         <script type="text/javascript">
+            function loadOtherPage(link) {
```
Comment (nlowe) [inline src/Athena/Views/Home/Home.cshtml]: What is this function for?

File: src/Athena/Views/Home/Home.cshtml [hunk 4]
```diff
@@ -47,14 +47,50 @@ else
...
+                $("<iframe class='printpage'>") // create a new iframe element
+                    .attr("src", link) // point the iframe to the page link you want to print
+                    .appendTo("body");
+            }
+            function ConfirmRemove(event) {
```
Comment (nlowe) [inline src/Athena/Views/Home/Home.cshtml]: This function isn't needed as far as I can tell.

File: src/Athena/Views/Home/Home.cshtml (hunk 3 above)
Comment (nchurchill) [reply src/Athena/Views/Home/Home.cshtml]: nothing at all

File: src/Athena/Views/Home/Home.cshtml (hunk 2 above)
Comment (nchurchill) [reply src/Athena/Views/Home/Home.cshtml]: done

File: src/Athena/Views/Home/Home.cshtml (hunk 1 above)
Comment (nchurchill) [reply src/Athena/Views/Home/Home.cshtml]: changed to remove-class-modal

File: src/Athena/Views/Home/Home.cshtml (hunk 4 above)
Comment (nchurchill) [reply src/Athena/Views/Home/Home.cshtml]: its not

**Pull request merged**